dotcom-rendering/src/components/Masthead/HighlightsCard.stories.tsx

```diff
@@ -105,3 +105,19 @@ export const WithStarRating: Story = {
 	parameters: {},
 	name: 'With Star Rating',
 };
+
+export const WithPodcastSeriesImage: Story = {
+	args: {
+		format: {
+			display: ArticleDisplay.Standard,
+			design: ArticleDesign.Standard,
```

Comment:
The story doesn't show the podcast image, I guess because this needs to be? 

```suggestion
			design: ArticleDesign.Audio,
```

Reply:
🤦  thank you!